pkgs/top-level/aliases.nix

```diff
@@ -572,6 +572,8 @@ mapAliases {
 
   # Julia
   julia_16-bin = throw "'julia_16-bin' has been removed from nixpkgs as it has reached end of life"; # Added 2024-10-08
+  julia_19-bin = throw "'julia_19-bin' has been removed from nixpkgs as it has reached end of life"; # Added 2024-12-02
```

Comment:
AFAICT Julia 1.9 is still maintained, do we actually need to remove it at this point?

It sounds like 1.6 has become unmaintained, but releases after that are still maintained:

https://discourse.julialang.org/t/julia-v1-11-0-has-been-released-and-v1-10-is-now-lts/121064

Reply:
It that case let's leave it around for a while more.